cpp/util/etcd.h

```diff
@@ -76,9 +81,10 @@ class EtcdClient {
 
   typedef std::function<void(const std::vector<Node>& updates)> WatchCallback;
 
-  // TODO(pphaneuf): This should take a set of servers, not just one.
   EtcdClient(UrlFetcher* fetcher, const std::string& host, uint16_t port);
 
+  EtcdClient(UrlFetcher* fetcher, const std::list<HostPortPair>& etcds);
```

Comment:
I'm almost tempted to say there should be just the one constructor... Could still use it fairly easily, with something like `{{host, port}}`, or `{make_pair(host, port)}`.